ports/argus/portfile.cmake

```
@@ -0,0 +1,30 @@
+vcpkg_download_distfile(ARCHIVE
+    URLS "https://github.com/lucocozz/Argus/releases/download/v${VERSION}/argus-${VERSION}-source.tar.gz"
+    FILENAME "argus-${VERSION}-source.tar.gz"
+    SHA512 cdb6f96ac0c80d9aaa9d1e1c3161549d4fd324ac755aa61f83070665f6cba9389e6f28791c41c9d0fce2f7c4bf7f2674e65b165cac648efac7fa502d312fa2da
+)
+
+vcpkg_extract_source_archive_ex(
+    OUT_SOURCE_PATH SOURCE_PATH
+    ARCHIVE ${ARCHIVE}
+)
```

```suggestion
vcpkg_from_github(OUT_SOURCE_PATH SOURCE_PATH
  REPO lucocozz/Argus
  REF "v${VERSION}"
  SHA512 <sha512>
  HEAD_REF main
)
```

Use `vcpkg_from_github` instead.

---

The contents of the auto-generated tar file are the same as the ones acquired by vcpkg_from_github using the tagged REF.